File: BaseLua/src/main/java/org/anime_game_servers/lua/models/MockLuaValue.java
package org.anime_game_servers.lua.models;


import org.anime_game_servers.lua.engine.LuaValue;

public abstract class MockLuaValue implements LuaValue {
    @Override
    public boolean isNull() {
        return false;
    }

    @Override
    public boolean isBoolean() {
        return false;
    }

    @Override
    public boolean isInteger() {
        return false;
    }

    @Override
    public boolean isLong() {
        return false;
    }

    @Override
    public boolean isDouble() {
        return false;
    }

    @Override
    public boolean isFloat() {
        return false;
    }

    @Override
    public boolean isString() {
        return false;
    }

    @Override
    public boolean isTable() {
        return false;
    }
}

File: BaseLua/src/main/java/org/anime_game_servers/lua/models/MockLuaValue.kt
package org.anime_game_servers.lua.models

import org.anime_game_servers.lua.engine.LuaValue

abstract class MockLuaValue : LuaValue {
    override fun isNull() = false
    override fun isBoolean() = false
    override fun isInteger() = false
    override fun isLong() = false
    override fun isDouble() = false
    override fun isFloat() = false
    override fun isString() = false
    override fun isTable() = false
}
